### DISCUSSION TITLE
Cannot activate license because there are 7 databases that use Revision compression while the new license doesn't include the usage of Revision compression show details

### DISCUSSION THREAD

**Discussion by luka-nikolic** [ravendb/ravendb#19655] [General]
Getting error when I paste regenerated licence:
`Cannot activate license because there are 7 databases that use Revision compression while the new license doesn't include the usage of Revision compression
show details`

I can't access database to disable compression since i can't close this modal:
![image](https://github.com/user-attachments/assets/81e39b65-02f8-4746-b6c5-145208bb6d7e)

**Comment by karmeli87** [comment]
Hi,

I'm afraid that the only option for you right now is to fix this via code.
You need to fetch all databases and disable the compression.

Something like that (note that I haven't tested the code below)
```
// Fetch all database names
var operation = new GetDatabaseNamesOperation(0, 25); // Adjust page size if needed
var databaseNames = await store.Maintenance.Server.SendAsync(operation);

Console.WriteLine("Found databases:");
foreach (var dbName in databaseNames)
{
  Console.WriteLine(dbName);

  try
  {
	  // Fetch current database record
	  var dbRecord = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(dbName));

	  // Modify the compression settings
	  dbRecord.DocumentsCompression = new DocumentsCompressionConfiguration
	  {
		  CompressRevisions = false,
		  CompressAllCollections = false
	  };

	  // Save the updated configuration
	  await store.Maintenance.Server.SendAsync(new UpdateDatabaseOperation(dbRecord, dbRecord.Etag));

	  Console.WriteLine($"Updated compression settings for database: {dbName}");
  }
  catch (Exception ex)
  {
	  Console.WriteLine($"Error updating database {dbName}: {ex.Message}");
  }
}
```

**Comment by luka-nikolic** [comment]
For anyone looking for a quick solution. I set modal element's css to display: none in devtools. Than i was able to enter and disable compression.
